content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "REANA 0.9.1 is released"
+date: 2023-09-28T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.1, a minor update that allows users to preview PDF and ROOT files from the web interface, makes it easier to clean up old workflows to free disk space, adds support for integrating with Keycloak, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### PDF/ROOT preview
+
+REANA 0.9.1 improves the preview of workspace files from the web interface by adding support for PDF and ROOT files.
+You will be able to interact with ROOT plots directly from your browser, without the need to download the files on your computer.
+
+![ROOT file preview](/images/reana-0.9.1-root-preview.png)
+
+### Pruning workspaces
+
+REANA already provides tools to keep disk usage under control, for example by setting up automatic retention rules or by deleting old workflows.
+REANA 0.9.1 also introduces the new `prune` command, which can be used to clean up your workflows by deleting all the files in their workspaces, with the exclusion of input and output files, so that you don't have to delete each file one by one.
+
+```console
+$ reana-client prune -w reana-demo-root6-roofit
+==> SUCCESS: The workspace has been correctly pruned.
+```
+
+Take a look to the related [Prune the workspace](https://docs.reana.io/advanced-usage/user-quotas/#prune-the-workspace) documentation page to learn more about this new command.
+
+### ... and much more!
+
+This new release brings many more small improvements. For example, it's now possible to stop workflows from the web UI, by clicking on the _Stop workflow_ button in the actions menu.
+Furthermore, restarted Snakemake workflows now report the correct total number of jobs, by excluding cached jobs that were simply reused from previous runs in the workspace and not really executed.
+
+## What's new for the administrators?
+
+### Keycloak
+
+REANA provides users with two ways to log in, either using username/password or via an external SSO provider.
```

Comment:
We are speaking to admins here, so perhaps:

You can configure your REANA deployment with two ways for user login, either using local user database with usernames and passwords or using an external Single Sign On provider. Up to now, only CERN SSO was supported. As of REANA 0.9.1, you can now integrate your deployment with any generic Keycloak identity and access management instance.

(and perhaps link to https://www.keycloak.org/ since people may not know what that is)